ACM_General/accounts/models.py

```diff
@@ -15,15 +15,15 @@
 
 class User(AbstractBaseUser):
     """
-     @Desc - Overloading of the base user class to enable email validation
-             as apposed to username validation in default django
-     @Fields -
-      id - 16 character UUID which uniquely identifies each user
-      email - Stores the user's email
-      date_joined - Stores when the user signs up
-      is_active - Whether or not a user accoutn should be considered 'active'
-      is_admin - Stores whether or not the user can access the admin panel
-      objects - Container for the User Manager
+    Overloading of the base user class to enable email validation
+    as apposed to username validation in default django.
+
+    id - 16 character UUID which uniquely identifies each user
```

Comment:
For these documentation attributes, use a `#: comment` above the variable itself for it to be processed by sphinx automatically. See Zack's documentation in `events` for an exmaple